### ISSUE TITLE
Alt text for thumbnails in search window is truncated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SmithKettlewellRERC/DVX-YouDescribe-issues#49]
```
What steps will reproduce the problem?
1. to to http://youdescribe.ski.org/search.php
2. use a screen reader to navigate through the table of search results.
3. note that only the first word of each video's title is announced when moving 
over the thumbnails on each row.

What is the expected output? What do you see instead?
It is expected that the ALT text would be identical to the title of the video 
in the second column. Instead of the entire title, you hear only the first word 
of the title.

What is the URL of the page causing the error?
http://youdescribe.ski.org/search.php


Please use labels and text to provideThe problem is definitely that there are 
no quotation marks around the Alt text. Alt tags should be 
alt="alt text to be spoken"
not
alt=alt text to be spoken

This is a simple fix.
```

Original issue reported on code.google.com by `joshua.a...@gmail.com` on 2 May 2013 at 6:48
